steps/keras/callbacks.py

```diff
@@ -34,6 +34,23 @@ def on_epoch_end(self, epoch, logs={}):
             K.set_value(self.model.optimizer.lr, self.gamma * K.get_value(self.model.optimizer.lr))
 
 
+class UnfreezeLayers(Callback):
```

Comment:
@jakubczakon class name suggests that you can unfreeze multiple layers. In the next iteration, think about parameterizing `from_layer` and `to_layer`.